<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
border-style:solid;
color:rgb(231, 255, 231);
border-color:rgba(205,205,205,0.8);/*边框颜色*/

}
/*鼠标按下状态*/
QPushButton:pressed
{
color:rgb(231, 255, 231);
background-color:rgba(198,198,220,06);/*背景色*/
border-style:solid;/*边框样式 solid实线 none无边框 inset/outset 3D边框*/
}
/*鼠标悬停状态*/
QPushButton:hover
{
color:rgb(231, 255, 231);
border-image: url(:/new/prefix1/media/media/stateMainMenu/menuHighlight.png);
}


</string>
   </property>
   <property name="text">
    <string>s t a r t</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/media/media/1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/media/media/2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/media/media/3.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/media/media/4.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/media/media/5.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/media/media/6.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/media/media/7.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-10</y>
     <width>601</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/media/media/stateMainMenu/mainMenuLogo.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>-10</y>
     <width>821</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/media/media/stateMainMenu/mainMenuBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
border-style:solid;
color:rgb(231, 255, 231);
border-color:rgba(205,205,205,0.8);/*边框颜色*/

}
/*鼠标按下状态*/
QPushButton:pressed
{
color:rgb(231, 255, 231);
background-color:rgba(198,198,220,06);/*背景色*/
border-style:solid;/*边框样式 solid实线 none无边框 inset/outset 3D边框*/
}
/*鼠标悬停状态*/
QPushButton:hover
{
color:rgb(231, 255, 231);
border-image: url(:/new/prefix1/media/media/stateMainMenu/menuHighlight.png);
}


</string>
   </property>
   <property name="text">
    <string>T w o  p l a y e r   V.S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
border-style:solid;
color:rgb(231, 255, 231);
border-color:rgba(205,205,205,0.8);/*边框颜色*/

}
/*鼠标按下状态*/
QPushButton:pressed
{
color:rgb(231, 255, 231);
background-color:rgba(198,198,220,06);/*背景色*/
border-style:solid;/*边框样式 solid实线 none无边框 inset/outset 3D边框*/
}
/*鼠标悬停状态*/
QPushButton:hover
{
color:rgb(231, 255, 231);
border-image: url(:/new/prefix1/media/media/stateMainMenu/menuHighlight.png);
}


</string>
   </property>
   <property name="text">
    <string>H e l p</string>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>pushButton1</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
